### ISSUE TITLE
Circuit Diagram

### ISSUE THREAD

**Issue by pandhersaab27** [BenjaminStienlet/ArduinoClock#2]
Hello again Benjamin
If possible for you can you please upload circuit diagram. Or maybe just post some more internal photos so I can see the circuitry.
And can you tell me what else would I need in terms of components these two are what I'm currently looking at see if this will work. 
http://leeselectronic.com/en/product/15296.html
http://leeselectronic.com/en/product/16085.html

I have a lot of capacitors and resistors in various capacities, what else will I need, I have to get it ready within a month so I want to plan and gather all the components beforehand.
Thank You very much for your support.

**Comment by BenjaminStienlet** [comment]
Here are some more images, this is all I have left. I can't find my circuit diagram anymore.

- [http://imgur.com/RTpYWvH](http://imgur.com/RTpYWvH)
- [http://imgur.com/63PGH25](http://imgur.com/63PGH25)
- [http://imgur.com/KxrVumE](http://imgur.com/KxrVumE)

You'll also need the following:

- Light-dependent resistor
- Large capacitor
- Buttons & switches
- Power supply & female plug

Look at the Adafruit guide to see how to connect the LEDs and estimate how much power you will need: [https://learn.adafruit.com/adafruit-neopixel-uberguide/basic-connections](https://learn.adafruit.com/adafruit-neopixel-uberguide/basic-connections)

**Comment by pandhersaab27** [comment]
Thank you, I'll review the information you provided.

**Comment by pandhersaab27** [comment]
Hi
I got my Trinket Pro 5V and was checking the specifications, so it provides only 150mA. I couldn't find if this was for 150mA for each pin or in total. So I wanted to check if the current from Trinket was enough to run all those LEDs or did you use something else?
Thank You again.

**Comment by BenjaminStienlet** [comment]
No, it is not enough. The power for the LEDs has to come straight from the power supply. The only connection from the Arduino to the LEDs is 1 data cable for each group.

**Comment by pandhersaab27** [comment]
Oh so for these LEDs I can control their On/Off state just with data pin even if its always connected to power? I still haven't received these LEDs so that why so much confusion.

**Comment by BenjaminStienlet** [comment]
Yes, that's how it works

**Comment by HaydenBrady** [comment]
Hey Ben, is there any chance that you've since found the circuit diagram? and if not if you could take a birds-eye view picture of your perfboard if you still have the clock? Thanks so much for the fun project idea and the code!

**Comment by BenjaminStienlet** [comment]
Hi Hayden, I haven't found the circuit diagram anymore. I'll try to take a picture when I'm home.

**Comment by HaydenBrady** [comment]
Thanks so much!!
